src/neuroconv/tools/spikeinterface/spikeinterface.py

```diff
@@ -203,11 +200,16 @@ def add_electrical_series_to_nwbfile(
     Deprecated. Call `add_recording_to_nwbfile` instead.
     """
 
+    warnings.warn(
+        "This function is deprecated and will be removed in or October 2025. "
+        "Use the 'add_recording_metadata_to_nwbfile' function instead.",
+        DeprecationWarning,
+    )
+
```

Comment:
Did you mean `add_recording_to_nwbfile` instead of `add_recording_metadata_to_nwbfile`?

Reply:
Yes. Thanks!